File: sdk/scala-sdk-protobuf/src/main/scala/kalix/scalasdk/workflow/WorkflowContext.scala
/*
 * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package kalix.scalasdk.workflow

import kalix.scalasdk.Context

trait WorkflowContext extends Context {

  /**
   * The id of the workflow that this context is for.
   *
   * @return
   *   The workflow id.
   */
  def workflowId: String
}

File: sdk/java-sdk-protobuf/src/main/java/kalix/javasdk/workflow/WorkflowContext.java
/*
 * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
 */

package kalix.javasdk.workflow;

import kalix.javasdk.Context;

public interface WorkflowContext extends Context {
  /**
   * The id of the workflow that this context is for.
   *
   * @return The workflow id.
   */
  String workflowId();
}
